tests/Fixtures/Kernel.php

```diff
@@ -36,6 +37,10 @@ public function registerBundles(): iterable
         if (\getenv('USE_DAMA_DOCTRINE_TEST_BUNDLE')) {
             yield new DAMADoctrineTestBundle();
         }
+
+        if ('migrate' === \getenv('FOUNDRY_RESET_MODE')) {
```

Comment:
I believe it's more safe to rely on `$_SERVER` or `$_ENV` (https://github.com/laravel/framework/pull/7425)

Reply:
I'll address in a separate PR (created issue #154 to track).